src/components/control/multi-autocompleter.jsx

```diff
@@ -64,12 +76,27 @@ const MultiAutocompleter = forwardRef(function MultiAutocompleter(props, ref) {
       });
     }
 
-    if (props.value.length) fetchPropsValue();
-  }, [props.value.map(props.optionIDGetter).join(',')]);
+    if (value.length) {
+      fetchPropsValue();
+    } else {
+      setValueForSelect([]);
+    }
+  }, [listOfIdsString(value), options]);
```

Comment:
Why does this re-fetch the selected options when it fetches the autocompleter dropdown choices?

Reply:
If value changes, so say a new array of ids, then it needs to re-fetch with the only query including those models in order to set the correct data structure.

Reply:
Given that statement, this effect is only dependent on the `value` changing:

```suggestion
  }, [listOfIdsString(value)]);
```